### ISSUE TITLE
interest in adding created, modified to task models.

### ISSUE THREAD

**Issue by jmerkow** [mback2k/django-celery-model#6]
I was going to add created and modified fields to the model task meta class.

i.e. something like 

```python
class ModelTaskMeta(models.Model):
    ...
    created = models.DateTimeField(auto_now_add=True, editable=False)
    modified = models.DateTimeField(auto_now=True)
```

**Comment by mback2k** [comment]
Sounds like a good idea. Could you prepare a corresponding pull request?

**Comment by mback2k** [comment]
Thanks a lot, closed #7 via e287975a8245138d71c0fd47338edb00b08b47fc.
